### DISCUSSION TITLE
how can I learn to use this software? any tutorial available?

### DISCUSSION THREAD

**Discussion by nishannazer** [manaakiwhenua/LUMASS#3] [Q&A]
Please help me use this software as I believe this allows coupling raster images to system dynamic models which will be useful for my research.

**Comment by heralex** [comment]
I am working on improving the documentation and am planning to provide some tutorials as well. However, for now I'd recommend reading the [Documentation](https://manaakiwhenua.github.io/LUMASS/docs/Documentation) and 'playing' with the [sample data and models](https://manaakiwhenua.github.io/LUMASS/assets/material/SampleData_v0.4.zip) ... BTW, the old [User Guide](https://manaakiwhenua.github.io/LUMASS/assets/material/LUMASS_UserGuide_v0.3.pdf) contains a little HowTo
 on creating a simple processing pipeline. There also some notes on [Creating and Executing Models](https://manaakiwhenua.github.io/LUMASS/docs/mod_createpipeline) in the [Documentation](https://manaakiwhenua.github.io/LUMASS/docs/Documentation) section.

Also, if you could give me some idea of what you want to model, I might be able to give you some pointers as to which [Model Components](https://manaakiwhenua.github.io/LUMASS/docs/mod_component_reference) you should look at and may be how to structure your model (at a high level).
